jina/executors/dump.py

```diff
@@ -0,0 +1,132 @@
+import os
+import sys
+from typing import Iterable, Tuple
+
+import numpy as np
+
+from jina.enums import BetterEnum
+
+
+class DumpTypes(BetterEnum):
+    """The enum of dump formats"""
+
+    DEFAULT = 0
+    NUMPY = 1
+
+
+BYTE_PADDING = 4
+
+
+class DumpPersistor:
```

Comment:
If it does not inherit from an Executor, it may need to be moved to some special folder. But I think these functions may belong in the Driver?

Reply:
Moved to the `drivers/`